<launch>
	<node pkg="nodelet" type="nodelet" name="mgr" args="manager"/>
	<node pkg="nodelet" type="nodelet" name="RosAriaNodelet" args="load RosAriaNodelet mgr"/>
</launch>